server/controllers/comment.controller.js

```diff
@@ -21,11 +21,13 @@ class CommentController {
   static async createComments(req, res) {
     const { id } = req.user;
     const { slug } = req.params;
-    const { comment } = req.body;
+    const { comment, highlightedtext } = req.body;
+
     const newcomment = await comments.create({
       userid: id,
       articleid: slug,
-      comment
+      comment,
+      highlightedtext,
```

Comment:
Good job. Can we use camelCasing instead for compound variables like `highlightedtext`. It will make the code easily readable.

Reply:
This is noted @cupidcoder, I will make the changes